webpack.common.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  entry: {
+    news: [
+      'babel-polyfill',
+      './src/app/App.js'
```

Comment:
It's better to move to constants. It will be difficult to support if config will be more complicated